### DISCUSSION TITLE
Continuously updated directory?

### DISCUSSION THREAD

**Discussion by doberbauer** [rikfair/photoshow#3] [Q&A]
Can this handle a folder that has photos periodically added to it? I'm doing a slideshow at my wedding and I have a script set up to download new photos added to a Google Drive folder I'm sharing out. How often does photoshow update its image cycle?

**Comment by doberbauer** [comment]
Nevermind! It _does_ continuously update. Thanks very much!

**Reply by rikfair** [reply]
Once it's running it will recreate the image list when it reaches the end of the current list and repeats. If you want it to repeat more often you can use the max_photos parameter, but that may cause some images to be shown again before others. Hope that helps and all the best for your wedding.

**Reply by doberbauer** [reply]
Terrific! Thank you very much. This repo is a life saver for me. I was building a slideshow script myself but Tkinter was arbitrarily rotating images at times and not scaling them at others times. Yours just _works_ and it was easy enough to install via pip. Do you have a ko-fi or anything I can donate to?

**Reply by rikfair** [reply]
That's fantastic to hear. I don't have ko-fi or anything similar, but thanks for asking.
